### ISSUE TITLE
Error with Laravel 10 

### ISSUE THREAD

**Issue by amenk** [iMi-digital/laravel-request-logger#13]
After upgrading to Laravel 10 we get the following error when using this package:

```
 php artisan migrate

   INFO  Running migrations.  

  2019_05_25_222854_change_request_log_entries_table ...................................................................................... 7ms FAIL

   Error 

  Call to undefined method Doctrine\DBAL\Connection::createSchemaManager()

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:1210
    1206▕     public function getDoctrineSchemaManager()
    1207▕     {
    1208▕         $connection = $this->getDoctrineConnection();
    1209▕ 
  ➜ 1210▕         return $connection->createSchemaManager();
    1211▕     }
    1212▕ 
```

**Comment by mansasidibe** [comment]
Hi,
Make sure to update the package's version in your **composer.json** file and run composer update to get the latest version of the package and its dependencies.

This error may occur when there is a mismatch between the version of Laravel and the version of the package you're using

**Comment by amenk** [comment]
@ronphobos Thank very much you for the advice. You mean `doctrine/dbal`, right? Did you eventually test this already and have a pull request?

**Comment by amenk** [comment]
fixed in release 4.0.2
